### DISCUSSION TITLE
Multiple assessment submissions

### DISCUSSION THREAD

**Discussion by mertkabadayii** [ls1intum/Artemis#7594] [Q&A] [answered]
Hello, is there a way for the Tutors to increase the number of simultaneously opened assessment submissions more than 10?

**Comment by MaisiKoleni** [comment, marked as answer]
I think only the admin can change this. Unless the logic for that changed significantly since I left the team, there is just one configuration option for everyone.

Do you really need to keep all assessments open? What hinders the completion/cancellation?

If there are open questions that need to be resolved first, contact your course instructors to find a solution. It should be in their interest to make the assessment as efficient and easy as possible.

**Reply by mertkabadayii** [reply]
Thanks for the explanation, i will contact with the admin and the instructors.
